philology/src/main/java/com/jcminarro/philology/PhilologyResources.kt

```diff
@@ -1,10 +1,27 @@
 package com.jcminarro.philology
 
+import android.annotation.TargetApi
+import android.content.res.AssetFileDescriptor
+import android.content.res.ColorStateList
+import android.content.res.Configuration
 import android.content.res.Resources
+import android.content.res.TypedArray
+import android.content.res.XmlResourceParser
+import android.graphics.Movie
+import android.graphics.Typeface
+import android.graphics.drawable.Drawable
+import android.os.Build.VERSION_CODES
+import android.os.Bundle
+import android.util.AttributeSet
+import android.util.DisplayMetrics
+import android.util.TypedValue
+import androidx.annotation.RequiresApi
+import java.io.InputStream
 
 @Suppress("DEPRECATION")
-internal class PhilologyResources(baseResources: Resources) :
-    Resources(baseResources.assets, baseResources.displayMetrics, baseResources.configuration) {
+internal class PhilologyResources(
+    private val baseResources: Resources
+) : Resources(baseResources.assets, baseResources.displayMetrics, baseResources.configuration) {
     private val resourcesUtil = ResourcesUtil(baseResources)
 
     override fun getText(id: Int): CharSequence = resourcesUtil.getText(id)
```

Comment:
![image](https://user-images.githubusercontent.com/1655758/71818995-3d00a680-3093-11ea-9ff6-5f0e1a816a40.png)

Can we have those two overridden as well?

Reply:
The `getString(id: Int, vararg formatArgs: Any): String` already uses the `getString(id:Int): String` that we handle. 

The `obtainTypedArray(id: Int)` can relate to #28. I have added a [comment](https://github.com/JcMinarro/Philology/issues/28#issuecomment-571128089) there.

Let's leave them as is for now.

Reply:
We should override `updateConfiguration()` method as well to keep the internal status of the `baseResources` 
![Captura de pantalla de 2020-01-07 16-37-58](https://user-images.githubusercontent.com/4047514/71907887-42322400-316d-11ea-9f08-49c0db4e464d.png)

Reply:
Sorry for the delay.
I have been investigating this issue, and in the end I could understand why it is happening.
The constructor of the `Resources` is calling to `updateConfiguration()` method before the "init block` is run on our `PhilologyResources` :man_facepalming: 
I think we can merge this PR and I will try to fix it in another PR